tests/unit/scripting.tcl

```diff
@@ -140,6 +140,25 @@ start_server {tags {"scripting"}} {
         } {*execution time*}
     }
 
+    test {EVAL - Active expiry can't happen during long script} {
+        r config set lua-time-limit 10
```

Comment:
this test doesn't really prove active expiry isn't triggered, but i'm not sure we can do any batter..

maybe two possible improvements could be:
1. either increase the cron hz, or make the test take longer (e.g. 200ms), so that in theory there's a better chance for active expiry to kick in.
2. use deferring client to run the script, and interact with redis while it's busy (both triggering BUSY errors, and also running some commands that actually succeed (or fail at a later stage in the pipeline, like SHUTDOWN).
3. maybe we can also use this opportunity to make sure eviction doesn't take place too.

Reply:
I can certainly make the script run for longer just to increase the chances of active expiry running if it ever does. And I can also add eviction tests. 

I am not sure if it is a good idea to run commands during the script as it doesn't really help triggering the active expiry. Also for evictions during the script run it doesn't help either because the command would get rejected upfront.

Reply:
One other issue I noticed is that this new test fails in the auto-run even though it consistently passes locally on my machine. I am not sure how to fix that.

Reply:
> I can certainly make the script run for longer just to increase the chances of active expiry running if it ever does. And I can also add eviction tests.

> I am not sure if it is a good idea to run commands during the script as it doesn't really help triggering the active expiry. Also for evictions during the script run it doesn't help either because the command would get rejected upfront.

nothing you do there will probably not help in triggering a bug (considering there isn't one to trigger), but it'll increase the chances that if something breaks one day, this test will be able to detect it.
i.e. we'll trigger both `beforeSleep`, as well as `serverCron` and even some of the checks in `processCommand`.
assuming they're all good, the test will pass, and if one day we'll introduce any bug in any of them, we want that test to detect that.